pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -781,6 +781,14 @@ type TiDBAccessConfig struct {
 	User string `json:"user,omitempty"`
 	// SecretName is the name of secret which stores tidb cluster's password.
 	SecretName string `json:"secretName"`
+	// Whether enable the TLS connection between the SQL client and TiDB server
+	// Optional: Defaults to nil
+	// +optional
+	TLSClient *TiDBTLSClient `json:"tlsClient,omitempty"`
+	// Specify a secret for client cert in backup/restore
+	// Optional: Defaults to <cluster>-tidb-client-secret
+	// +optional
+	TLSSecret string `json:"tlsSecret,omitempty"`
```

Comment:
This should be included in the `TLSClient` field.

Reply:
done